### PR TITLE
Remove cluster issuers, bump to 1.0.0-rc.1

### PR DESCRIPTION
**Pull request by brooksmtownsend**
- **chore(*)!: remove cluster_seed/cluster_issuers**
- **chore: update CONTRIBUTING.md**
- **chore(wasmcloud): bump to 1.0.0-rc.1**

## Feature or Problem
This PR removes the cluster_seed and cluster_issuers command line flags (and internal plumbing), and takes the chore to bump to wasmCloud 1.0.0-rc.1 when we're ready :rocket:

## Related Issues
See #1220 for the motivation behind cluster seed and cluster issuers being removed in favor of response signing verification.

## Consumer Impact
Because the `cluster_issuers` field in the HostData struct for providers is _not_ something that serde can deserialize if it's missing, we will pass an empty vec until we can ensure that we can release a new version of all of our capability providers that don't depend on this field being passed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -184,13 +184,10 @@ that we're already aware of. It is also worth asking on the Slack channels.
...
 follow SemVer with a high degree of rigor. 
 
-As a pre-1.0 project, we are likely to have breaking changes on each release. However, these changes
-must be _clearly documented_ in the release notes for each release. Once we release a 1.0 version of
-wasmCloud, we will maintain a strong commitment to backward compatibility. All of our changes to
-protocols and formats will be backward compatible from one minor release to the next. No features,
-flags, commands, or APIs will be removed or substantially modified without a major version release
-(unless absolutely needed to fix a security issue). This often means that we have to tell people
-"no" or "wait" in order to preserve backward compatibility.
+As a post-1.0 project, wasmCloud maintains a strong commitment to backward compatibility. All of our
```
Comment (thomastaylor312) [inline CONTRIBUTING.md:187]: 😍

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -184,13 +184,10 @@ that we're already aware of. It is also worth asking on the Slack channels.
...
+changes to protocols and formats will be backward compatible from one minor release to the next. No
+features, flags, commands, or APIs will be removed or substantially modified without a major version
+release(unless absolutely needed to fix a security issue). This often means that we have to tell people
```
Comment (thomastaylor312) [inline CONTRIBUTING.md:190]: What do we have to tell people?

**Review by ricochet — APPROVED** [review]

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
One small docs change needs to happen but otherwise good

File: CONTRIBUTING.md (hunk 2 above)
Comment (brooksmtownsend) [reply CONTRIBUTING.md:190]: That the bird is the word

**Review by thomastaylor312 — APPROVED** [review]

**Pull request merged**